ugs-core/src/com/willwinder/universalgcodesender/i18n/Localization.java

```diff
@@ -56,18 +55,20 @@ synchronized public static boolean initialize(String language) {
     
     /**
      * Loads a given language.
-     * @param language
+     * @param language the language to load, ex: en, sv, de
+     * @param region the region of the language to load, ex: US, SE, DE
      * @return Returns false if some keys are missing compared to "en_US"
      */
     synchronized public static boolean initialize(String language, String region) {
         Localization.region = region;
-        locale = new Locale(language, region);
+        Locale locale = new Locale(language, region);
+        Locale.setDefault(locale);
         bundle = ResourceBundle.getBundle("resources.MessagesBundle", locale);
         return getKeyCount(bundle) >= getEnglishKeyCount();
     }
 
     public static String loadedLocale() {
-        return locale + "";
+        return Locale.getDefault() + "";
```

Comment:
Thanks, I understand now, I thought this was changing the UGS localization based on the system locale. I didn't realize the `Locale` defaults worked this way.